extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -30,13 +30,13 @@ var expectInteractionDetailsToMatch = async function(elem) {
   ).toBe(true);
   // Testing editor's value in default state.
   expect(
-    await objects.AlgebraicExpressionEditor(elem.element(by.tagName(
+    await objects.MathEditor(elem.element(by.tagName(
       'oppia-interactive-algebraic-expression-input'))).getValue()
```

Comment:
This is still `algebraic-expression-input`

Reply:
Sorry, is it supposed to be something else?

Reply:
Nope, I just didn't understand that `MathEditor` was working for both interactions. Thanks for explaining!